src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -142,7 +133,14 @@ export class CalciteDropdownItem {
     const contentEl = !this.href ? (
       slottedContent
     ) : (
-      <a {...attributes} class="dropdown-link" ref={(el) => (this.childLink = el)}>
+      <a
+        class="dropdown-link"
+        href={this.href}
+        ref={(el) => (this.childLink = el)}
+        rel={this.el.getAttribute("rel")}
+        target={this.el.getAttribute("target")}
+        title={this.el.getAttribute("title")}
```

Comment:
@driskull should we make explicit props for these as well?

Reply:
Yes 👍 

For props that already exist at global level like 'title' we will need to use label or something.

Reply:
explicit props means that they are documented and supported. Unlike before :)

Reply:
I matched what we just did with `calcite-button` and created a label prop to apply to `aria-label` on the `<a>` and ditch setting the `title` attribute.